Makefile

```
@@ -6,12 +6,30 @@ DIRS	= test bin locale src
 PYDIRS	= quipucords
 PIP_COMPILE_ARGS = --no-upgrade
 BINDIR  = bin
-PARALLEL_NUM ?= $(shell python -c 'import multiprocessing as m;print(int(max(m.cpu_count()/2, 2)))')
+PARALLEL_NUM ?= $(shell $(PYTHON) -c 'import multiprocessing as m;print(int(max(m.cpu_count()/2, 2)))')
 TEST_OPTS := -n $(PARALLEL_NUM) -ra -m 'not slow' --timeout=15
 
 QUIPUCORDS_CONTAINER_TAG ?= quipucords
 QUIPUCORDS_UI_PATH ?= ../quipucords-ui
 QUIPUCORDS_UI_RELEASE ?= latest
+GITHUB_API_TOKEN_SECRET ?= /run/secrets/gh_api_token
+GITHUB_API_TOKEN := $(file < $(GITHUB_API_TOKEN_SECRET))
+ifneq ($(GITHUB_API_TOKEN),)
+	GITHUB_API_AUTH = -H "Authorization: Bearer $(GITHUB_API_TOKEN)"
+endif
+
+ifndef DOCKER_HOST
+	PODMAN_SOCKET := $(shell podman machine inspect --format '{{.ConnectionInfo.PodmanSocket.Path}}' 2> /dev/null)
+	ifneq ($(.SHELLSTATUS), 0)
+		PODMAN_SOCKET := $(shell podman info --format '{{.Host.RemoteSocket.Path}}' 2> /dev/null)
+	endif
```

```suggestion
	PODMAN_SOCKET := $(shell podman machine inspect --format '{{.ConnectionInfo.PodmanSocket.Path}}' 2> /dev/null || podman info --format '{{.Host.RemoteSocket.Path}}' 2> /dev/null)
```

…due to issue I discussed in Slack here: https://redhat-internal.slack.com/archives/C02QSNF1UKE/p1695931278695499?thread_ts=1695910544.300239&cid=C02QSNF1UKE

But we need to verify that this version also works on podman _without_ a machine VM running. 😕

---

I took a slightly different version of this approach, with additional `|| echo -n ""`. The idea is that if you don't have podman installed, then PODMAN_SOCKET will be empty, and next conditional will be skipped.

On machine without VM, `podman machine inspect` returns 125 and reports that VM is missing. So it will fall back to `podman info`.

One problem I noticed is that if you have podman machine set up, but VM is shut down, `podman machine inspect` will still report socket, which does exist. So it will take VM path, and all tests are going to fail. Since macOS users are likely going to have VM started (or will start it when they notice all tests failing), and Linux users don't really have a reason to use podman with VM, I think it's alright to just leave it as it is.